content/courses/native-onchain-development/program-state-management.md

```diff
@@ -554,11 +554,10 @@ We're ready to build and deploy our program!
 You can test your program by submitting a transaction with the right instruction
 data. For that, feel free to use
 [this script](https://github.com/Unboxed-Software/solana-movie-client) or
-[the frontend](https://github.com/Unboxed-Software/solana-movie-frontend) we
-built in the
-[Deserialize Custom Instruction Data lesson](deserialize-custom-data). In both
-cases, make sure you copy and paste the program ID for your program into the
-appropriate area of the source code to make sure you're testing the right
+[the frontend](https://github.com/EmekaManuel/movie-review-dapp) we built in the
+[Deserialize Custom Instruction Data lesson](/content/courses/native-onchain-development/deserialize-custom-data-frontend.md).
+In both cases, make sure you copy and paste the program ID for your program into
+the appropriate area of the source code to make sure you're testing the right
 program.
 
 If you use the frontend, simply replace the `MOVIE_REVIEW_PROGRAM_ID` in both
```

Comment:
Having the same constant in two places is lame. Can you please fix that in the https://github.com/solana-developers/movie-review-frontend?

Reply:
> Having the same constant in two places is lame. Can you please fix that in the https://github.com/solana-developers/movie-review-frontend?

fixed ✅